jax/numpy/lax_numpy.py

```diff
@@ -3813,6 +3813,16 @@ def searchsorted(a, v, side='left', sorter=None):
   return _searchsorted(a, v, side)
 
 
+@_wraps(onp.digitize)
+def digitize(x, bins, right=False):
+  side = 'right' if not right else 'left'
+  return lax.cond(
+    len(bins) < 2 or bins[1] > bins[0],
```

Comment:
This is really interesting: usually `or` doesn't work in traced code (e.g. we have to write `jit(lambda x, y: x | y)` instead of `jit(lambda x, y: x or y)`), but here because `len(bins) < 2` is a trace-time constant, it does work!

Reply:
yay for short-circuit operators!